lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -332,11 +339,6 @@ def _parse_genbank(self, genome_obj):
         genome = {
             "id": params['genome_name'],
             "original_source_file_name": os.path.basename(genome_obj.consolidated_file),
-            "assembly_ref": genome_obj.assembly_ref,
-            "gc_content": genome_obj.gc_content,
-            "dna_size": genome_obj.dna_size,
-            "md5": genome_obj.md5,
-            "genbank_handle_ref": genome_obj.handle_service_output['handle']['hid'],
```

Comment:
all of this other than the handle ref should be known at the point this method is called, right?

Reply:
this method you mean `_parse_genbank(self, genome_obj)`?

Reply:
yeah.. although if you're pulling it from the assembly object then you won't know it until after the save if a ref isn't provided. Is that what's happening?

Reply:
right, `_save_assemblies` is after _parse_genbank now. So I add these attributes back in `_update_genome_data` right before `_save_genomes` step

Reply:
:+1: